rust/src/ffi/hashing.rs

```diff
@@ -24,10 +24,6 @@ use std::os::raw::c_char;
 pub const SC_SHA1_LEN: usize = 20;
 pub const SC_SHA256_LEN: usize = 32;
 
-// Length of hex digests without trailing NUL.
-pub const SC_MD5_HEX_LEN: usize = 32;
-pub const SC_SHA256_HEX_LEN: usize = 64;
```

Comment:
Would rather see if we can remove this from `src/rust.h` instead, and have these exported to C.

Reply:
Well, they are already exported to C, so we can remove them from rust.h no problem ✌️